docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -206,6 +209,23 @@ You can omit any of the operands of the `..` operator to obtain an open-ended ra
 
 :::code language="csharp" source="snippets/shared/MemberAccessOperators.cs" id="RangesOptional":::
 
+The following table shows various ways to express collection ranges:
+
+| Range operator expression | Description                                                                          |
+|---------------------------|--------------------------------------------------------------------------------------|
+| `..`                      | All values in the collection.                                                        |
+| `..{end}`                 | Values from the start to the `{end}` exclusively.                                    |
```

Comment:
@IEvangelist do you want all the descriptions to follow the same pattern?
If not, then for the cases with only one boundary specified, the description can be different. For example, here: 

> First `{end}` values of the collection.

`a..`:

> All values in the collection except the first `a` values

Said that, using the same pattern for all rows may be beneficial as well.

Reply:
I'm not sure, but I'm open to whatever you and @BillWagner think would make the most sense. I initially leaned towards the same pattern for the descriptions, as I wanted the distinction to be obvious from the wording alone. Which way do you prefer?

Reply:
@IEvangelist @BillWagner what about combining both approaches:

> Values from the start to the `{end}` exclusively. That is, first `{end}` values of the collection.

Reply:
Pushed changes, are we all good with this?

Reply:
@IEvangelist I have no more comments, all is fine to me. Thank you for this PR, I think both the table and examples greatly improve and clarify this section of the docs!